client/src/components/ChapterCard.tsx

```diff
@@ -1,30 +1,57 @@
 import React from 'react';
-import { Heading, Tag, Text, VStack } from '@chakra-ui/react';
+import {
+  Stack,
+  Heading,
+  Box,
+  Center,
+  Image,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
 import { Link } from 'chakra-next-link';
 import { Chapter } from 'generated/graphql';
 
-import { Card } from './Card';
-import { truncate } from '../helpers/truncate';
-
 type ChapterCardProps = {
   chapter: Pick<
     Chapter,
-    'id' | 'name' | 'description' | 'category' | 'details'
+    'id' | 'name' | 'description' | 'category' | 'details' | 'image'
   >;
 };
 
 export const ChapterCard: React.FC<ChapterCardProps> = ({ chapter }) => {
   return (
-    <Card w="full">
-      <VStack align="flex-start">
-        <Heading size="md" as="h2">
-          <Link href={`/chapters/${chapter.id}`}>{chapter.name}</Link>
-        </Heading>
-        <Text>{truncate(chapter.description, 120)}</Text>
-        <Tag>{chapter.category}</Tag>
-
-        {/* <Text>{truncate(chapter.details, 120)}</Text> */}
-      </VStack>
-    </Card>
+    <Center m={2}>
+      <Link href={`/chapters/${chapter.id}`} _hover={{}}>
+        <Box
+          w={'full'}
+          bg={useColorModeValue('white', 'gray.800')}
+          rounded={'md'}
+          overflow={'hidden'}
+        >
+          <Image
+            h={'168px'}
```

Comment:
Is there a good reason to use `{'168px'}` over `"168px"`?  To me, the former suggests "thing to be evaluated" and the latter just "thing".

Obviously it's not a big deal, but just quotes seems preferable and is consistent with the rest of the codebase.

Reply:
Double-quote was the common style in earlier days. It seems single quote became popular with HTML 5, so it could be an old habit for someone.

Reply:
I'm not quite sure *why*, but eslint is enforcing the double quotes... but only for react :man_shrugging:

I've been meaning to separate eslint and prettier.  When I do that, I'll try and figure out why it's being inconsistent with quotes.